{"name": "test_div[failed]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 3.3333333333333335 == 0.3", "trace": "self = <testing.test_calc.TestCalc object at 0x7f9c96a72b20>, c = 10, d = 3, expect = 0.3\n\n    @pytest.mark.parametrize(\n    \t\"c,d,expect\",\n    \tdiv_data, ids=div_ids\n    )\n    def test_div(self, c, d, expect):\n    \tresult = self.calc.div(c,d)\n>   \tassert result == expect\nE    assert 3.3333333333333335 == 0.3\n\ntest_calc.py:35: AssertionError"}, "attachments": [{"name": "stdout", "source": "3df9293c-b3b9-4db8-acb6-06433992d03e-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "c", "value": "10"}, {"name": "d", "value": "3"}, {"name": "expect", "value": "0.3"}], "start": 1621952176543, "stop": 1621952176543, "uuid": "9bb2fd35-e2cc-417c-89fb-128a8c66d556", "historyId": "a489c2561226748904bc8930ccd3bcfd", "testCaseId": "4a57f615ae930c12d705a9fc7a66b88d", "fullName": "test_calc.TestCalc#test_div", "labels": [{"name": "suite", "value": "test_calc"}, {"name": "subSuite", "value": "TestCalc"}, {"name": "host", "value": "MacBook-Pro.local"}, {"name": "thread", "value": "49750-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_calc"}]}